.github/CODEOWNERS

```
@@ -1,7 +1,28 @@
-
-*                       @Cray-HPE/docs-csm-reviewers
-
-background/ncn_*        @Cray-HPE/metal
-install/*               @Cray-HPE/metal
-install/livecd          @Cray-HPE/metal
-operations/network      @Cray-HPE/management-network
+*                                                       @Cray-HPE/docs-csm-reviewers
+background/ncn_*                                        @Cray-HPE/metal
+install/*                                               @Cray-HPE/metal
+install/livecd                                          @Cray-HPE/metal
+operations/CSM_product_management                       @Cray-HPE/cms-core-cfs
```

Found the other owner we should use for that one

```suggestion
operations/CSM_product_management                       @Cray-HPE/cms-core-cfs
operations/CSM_product_management                       @Cray-HPE/cms-core-csm-product
```